lib/runner.rb

```diff
@@ -0,0 +1 @@
+require "./game"
```

Comment:
@amymoller I wouldn't put the runner file inside `/lib`. Lib is for class definitions / method definitions, etc. The runner should just live in the main project folder (not in lib or test bc it doesn't fit into these categories) !